packages/iframe-wallet-poc/iframe-wallet/index.js

```diff
@@ -133,14 +133,14 @@ async function pickUser(pkey) {
 
 function getApps() {
   const openChannelAddress = Object.keys(
-    iframeWallet.currentUser.vm.cfState.channelStates
```

Comment:
This one was particularly silly, having a *cf*state on an obviously CF-branded object